### DISCUSSION TITLE
如何关闭候选符号的全角/半角提示？

### DISCUSSION THREAD

**Discussion by destroy314** [rime/home#1224] [Q&A]
![2023-02-26_05-45-18](https://user-images.githubusercontent.com/35775292/221381216-296395db-4e5e-43d7-a717-cd4ad701edb9.jpg)

**Comment by hbghlyj** [comment]
相关：[punctuator.cc#L196](https://github.com/rime/librime/blob/ab586ca1f32477edc00dfd6cfa555dd5425d283f/src/rime/gear/punctuator.cc#L196)
```
      "\xe3\x80\x94\xe5\x8d\x8a\xe8\xa7\x92\xe3\x80\x95";  // 〔半角〕
```

**Comment by hbghlyj** [comment]
相关：[CHANGELOG](https://github.com/rime/weasel/blob/81afe6c1042bcfa3ea59199846d75d779a2ddce7/CHANGELOG.md?plain=1#L578)
```
* 改進：標點符號註明〔全角〕〔半角〕
```
